lib/system/schubfach.nim

```diff
@@ -0,0 +1,451 @@
+##  Copyright 2020 Alexander Bolz
```

Comment:
can you add a readme.md somewhere to show:
* the git repo involved, the files that were copied, and the hash at which the repo was cloned
* the c2nim command used (i'm assuming smthg like `c2nim --cpp --keepBodies -o:lib/system/schubfach.nim $Drachennest/src/schubfach_32.cc`)
* ideally, split in 2 commits, 1 with raw c2nim output (maybe marked `[wip]` so git bisect can learn to skip it), and one with the manual modifications, that way it's more reproducible/transparent and easier to update from upstream

Reply:
Good ideas, will do it maybe later. But I consider remaining bugs to be our own, not related to dragonbox/schubfach, just like we avoided certain TLSF bugs by porting over to Nim.

Reply:
i was referring to not just bugs but performance improvements, not that they can't be ported over again

what do you mean TLSF bugs ?